modules/MIPS/generate_MIPS_xml.php

```diff
@@ -1,69 +1,57 @@
+<html>
+<body style="background-color:#fedf98;">
 <?php
 /**
  * Clinical Measures Engine Report Form
  * Copyright (C) 2015 - 2017      Suncoast Connection
- * 
+ *
  * LICENSE: This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0
- * See the Mozilla Public License for more details. 
+ * See the Mozilla Public License for more details.
  * If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
- * 
+ *
  * @author  Art Eaton <art@suncoastconnection.com>
  * @author  Bryan lee <leebc@suncoastconnection.com>
- * @package LibreHealthEHR 
+ * @package LibreHealthEHR
  * @link    http://suncoastconnection.com
  * @link    http://librehealth.io
```

Comment:
@Priyanka488 you can add your name in here too 😉

Reply:
Okay !! : )